scripts/consensus.vcf

```diff
@@ -0,0 +1,528 @@
+##fileformat=VCFv4.2
+##FILTER=<ID=PASS,Description="All filters passed">
+##INFO=<ID=MQ,Number=1,Type=Integer,Description="RMS mapping quality (normal sample)">
+##INFO=<ID=MQ0,Number=1,Type=Integer,Description="Number of MAPQ=0 reads (normal sample)">
+##INFO=<ID=CAL,Number=.,Type=String,Description="List of callers making this call">
+##INFO=<ID=Hotspot,Number=0,Type=Flag,Description="Included by exception to consensus rule due to "hotspot" status">
+##FORMAT=<ID=GT,Number=1,Type=String,Description="Consensus genotype">
+##FORMAT=<ID=AD,Number=R,Type=Integer,Description="Consensus depths for the ref and alt alleles in the order listed">
+##FORMAT=<ID=AF,Number=A,Type=Float,Description="Consensus allele frequency">
+##FORMAT=<ID=DP,Number=1,Type=Integer,Description="Consensus depth">
+##FORMAT=<ID=GTC,Number=.,Type=String,Description="Genotypes for Strelka2|Mutect2|VarDict|Lancet">
+##FORMAT=<ID=GT_STATUS,Number=.,Type=String,Description="Degree of unanimity of genotype: 'unanimous', 'majority', or 'deadlock' if equally supported by individual callers">
+##FORMAT=<ID=ADC,Number=.,Type=String,Description="Allele depths for Strelka2|Mutect2|VarDict|Lancet">
+##FORMAT=<ID=DPC,Number=.,Type=String,Description="Read depths for Strelka2|Mutect2|VarDict|Lancet">
+##FORMAT=<ID=AFC,Number=.,Type=String,Description="Allele frequencies for Strelka2|Mutect2|VarDict|Lancet">
+##FORMAT=<ID=ADR,Number=R,Type=Integer,Description="Difference between highest and lowest AD">
+##FORMAT=<ID=AFR,Number=A,Type=Float,Description="Difference between highest and lowest AF">
+##FORMAT=<ID=DPR,Number=1,Type=Integer,Description="Difference between highest and lowest DP">
+##contig=<ID=chr1,length=248956422>
+##contig=<ID=chr2,length=242193529>
+##contig=<ID=chr3,length=198295559>
+##contig=<ID=chr4,length=190214555>
+##contig=<ID=chr5,length=181538259>
+##contig=<ID=chr6,length=170805979>
+##contig=<ID=chr7,length=159345973>
+##contig=<ID=chr8,length=145138636>
+##contig=<ID=chr9,length=138394717>
+##contig=<ID=chr10,length=133797422>
+##contig=<ID=chr11,length=135086622>
+##contig=<ID=chr12,length=133275309>
+##contig=<ID=chr13,length=114364328>
+##contig=<ID=chr14,length=107043718>
+##contig=<ID=chr15,length=101991189>
+##contig=<ID=chr16,length=90338345>
+##contig=<ID=chr17,length=83257441>
+##contig=<ID=chr18,length=80373285>
+##contig=<ID=chr19,length=58617616>
+##contig=<ID=chr20,length=64444167>
+##contig=<ID=chr21,length=46709983>
+##contig=<ID=chr22,length=50818468>
+##contig=<ID=chrX,length=156040895>
+##contig=<ID=chrY,length=57227415>
+##contig=<ID=chrM,length=16569>
+##INFO=<ID=END,Number=1,Type=Integer,Description="Stop position of the interval">
```

Comment:
Hmmm, this is interesting. Where does it come from? Doesn't seem to be added in the code...

Reply:
I'm not sure! It doesn't seem to be automatically added when you create a VCF, just the version and filter PASS are added to the header.

Reply:
Um, hmmm, I don't want to hold up this PR on this, but maybe while you are working on the performance boost, see if you can dump this guy. Like, if this field is in INFO, delete...